website/frontend/src/services/ServerRPCRequestService.ts

```diff
@@ -24,18 +24,12 @@ type RPCRequest = {
   params?: Object;
 };
 
-type RPCError = {
-  code: string | JSONRPCErrorCode;
-  message: string;
-  data: string;
-};
-
 // eslint-disable-next-line @typescript-eslint/naming-convention
 type _RPCResponse<Result> = {
   id: JsonRpcId;
   jsonrpc: JsonRpcVersion;
   result: Result;
-  error: RPCError;
+  error: string;
```

Comment:
Its not by specification https://www.jsonrpc.org/specification#response_object

Reply:
Thanks, gonna be resolved on backend.

Reply:
Fixed.